From af32bb80fc317cce4a7462876833999ffce3434d Mon Sep 17 00:00:00 2001
From: Hong H. Pham <hong.pham@windriver.com>
Date: Wed, 5 Nov 2008 11:41:40 -0500
Subject: [PATCH] Add arch/sparc64/configs/ directory

The arch/sparc64/configs/ directory has been removed in recent kernel releases.
This causes the WRLinux kernel build process to fail at the kernel
configuration phase, as the build scripts looks for the board's default kernel
configuration file in
arch/sparc64/configs/../../../wrs/cfg/<BSP>/<BSP>-config-<KERNEL VERSION>

This patch creates the arch/sparc64/configs/ directory so that the WRLinux
kernel build process can find the board's default kernel configuration file.

Signed-off-by: Hong H. Pham <hong.pham@windriver.com>
---
 arch/sparc64/configs/.keepme |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/sparc64/configs/.keepme b/arch/sparc64/configs/.keepme
new file mode 100644
index 0000000..8d1c8b6
--- /dev/null
+++ b/arch/sparc64/configs/.keepme
@@ -0,0 +1 @@
+ 
-- 
1.5.5.1

